c/cxf/cxf_ubi_8.5.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+###!/bin/bash -e
```

Comment:
Hi @vinodk99 - is this change intentional?

Reply:
@kandarpamalipeddi - Can you please comments on it. Seems it is your commit.